agent/util-scripts/pbench-tool-meister-client

```diff
@@ -71,11 +76,13 @@ def main(argv):
     redis_server = os.environ.get("PBENCH_REDIS_SERVER", f"localhost:{def_redis_port}")
     parts = redis_server.split(":", 1)
     if len(parts) != 2:
-        raise Exception("Bad Redis server specified, '{redis_server}'")
+        logger.error("Bad Redis server specified, {!r}", redis_server)
```

Comment:
Why the `!r` directive?  `redis_server` is a `string`, right?  Is this just a trick for adding quotation marks to the output?  (You didn't use this trick for `action` at line 65....)

Reply:
Sort of, it renders the `repr` version of the object, so that if it is not a string, we see that in the exception string.

Reply:
`os.environ.get()` in this specific case returns a `string`, so `repr`, as far as I can tell, just adds single quotes...which is kind of cute, but just a little weird.

Reply:
So I want to maintain the discipline of using `repr` in these cases so that when things change (link when I changed from having a `string` to a `Path` and I got errors) the `repr` would have shown me that, but forced "stringification" means that I never see the actual difference.